data/src/main/java/com/example/data/Data.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.data
+
+import android.util.Log
+import com.example.domain.Base
+import com.example.domain.Item
+import com.google.gson.GsonBuilder
+import retrofit2.*
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Data {
+    var repo: MutableMap<String, List<Base>> = mutableMapOf()
+
+    private fun loadData(item: String) {
+        var retrofit: Retrofit = Retrofit.Builder()
+            .baseUrl("https://raw.githubusercontent.com/karl-park/com.ninetynine.healthysalad/master/server/")
```

Comment:
the base url is a constant, right? then you have to extract it to the external variable.